script.xbmcbackup/resources/lib/tinyurl.py

```diff
@@ -0,0 +1,12 @@
+# this is duplicated in snipppets of code from all over the web, credit to no one
+# in particular - to all those that have gone before me!
+from future.moves.urllib.request import urlopen
```

Comment:
Are you sure this works in python3?

Reply:
Yes, I just tested it locally and it worked. Using a nightly build where the Python version is listed as 3.7.1.